packages/devtools_app/test/shared/extent_delegate_list_view_test.dart

```diff
@@ -14,7 +14,7 @@ void main() {
 
     Future<void> pumpList(
```

Comment:
nit maybe rename to `wrapAndPump` now that the widget passed in will not always be a ListView

Reply:
Sure thing!